app/main.py

```diff
@@ -16,4 +17,19 @@
 }
 collection_of_coins = {1, 2, 25}
 
-# write your code here
+sorted_variables = {
```

Comment:
In this exercise, you must fill in this dictionary without using additional code